src/Storage/Migration/v0.00/01-setup-tables.sql

```diff
@@ -74,7 +74,10 @@ CREATE INDEX IF NOT EXISTS instances_partyid ON storage.instances(partyId);
 CREATE INDEX IF NOT EXISTS instances_appid ON storage.instances(appId);
 CREATE INDEX IF NOT EXISTS instances_appid_taskId ON storage.instances(appId, taskId);
 CREATE INDEX IF NOT EXISTS instances_appid_lastchanged ON storage.instances(appId, lastChanged);
+CREATE INDEX IF NOT EXISTS instances_partyid_lastchanged ON storage.instances(partyId, lastChanged);
```

Comment:
editing an existing migration version will require manual intervention in those environments where db has been deployed previously

Reply:
Ref. comment in PR. This is done deliberately to make it easier to read, maintain and compare db changes until conversion. The conversion requires manual operations anyway.

Reply:
I share the concern and are afraid we might forget something along the way if too much is to be handled manually.

Reply:
Let's discuss the conversion approach.